src/managers/GuildApplicationCommandManager.js

```diff
@@ -119,6 +119,86 @@ class GuildApplicationCommandManager extends ApplicationCommandManager {
     );
   }
 
+  /**
+   * Add permissions to a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {ApplicationCommandPermissionData[]} permissions The permissions to add to this command
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Block a role from the command permissions
+   * guild.commands.addPermissions('123456789012345678', [
+   *   {
+   *     id: '876543211234567890',
+   *     type: 'ROLE',
+   *     permission: false
+   *   },
+   * ])
+   *   .then(console.log)
+   *   .catch(console.error);
+   */
+  async addPermissions(command, permissions) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    const perms = await this.fetchPermissions(id).catch(err => {
+      if (err.httpStatus !== 404) throw err;
+      return [];
+    });
+
+    const newPermissions = [...permissions];
+    for (const perm of perms) {
+      if (!newPermissions.find(x => x.id === perm.id)) {
+        newPermissions.push(perm);
+      }
+    }
+
+    return this.setPermissions(id, newPermissions);
+  }
+
+  /**
+   * Remove permissions from a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {UserResolvable | RoleResolvable | UserResolvable[] | RoleResolvable[]} userOrRole The user(s) and role(s)
+   * to remove from the command permissions
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Remove a user permission from this command
+   * guild.commands.addPermissions('123456789012345678', '876543210123456789')
+   *   .then(console.log)
+   *   .catch(console.error);
+   * @example
+   * // Remove multiple roles from this command
+   * guild.commands.removePermissions('123456789012345678', [ '876543210123456789', '765432101234567890' ])
+   *    .then(console.log)
+   *    .catch(console.error);
+   */
+  async removePermissions(command, userOrRole) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    let permissions = await this.fetchPermissions(id).catch(err => {
+      if (err.httpStatus !== 404) throw err;
+      return [];
+    });
+
+    if (Array.isArray(userOrRole)) {
+      userOrRole = userOrRole
+        .map(x => this.client.users.resolveID(x) ?? this.guild.roles.resolveID(x))
+        .filter(x => !!x);
```

Comment:
I believe this should throw if an ID were to not be resolveable, as opposed to silently failing, similar to what [BitField does](https://github.com/discordjs/discord.js/blob/master/src/util/BitField.js#L151).